src/background/extension.ts

```diff
@@ -185,6 +186,10 @@ export class Extension {
         };
     }
 
+    async loadConfig(options: {local: boolean}) {
```

Comment:
I don't see why this method is needed, `this.loadConfig()` is about the same as `this.config.load()`